docs/extend/extend-apiml/api-medation-sso-integration-extenders.md

```diff
@@ -11,15 +11,17 @@ This article does not cover the client methods to call API ML and authenticate.
 
 To integrate with API Mediation Layer and leverage Single Sign On, choose from the following three possible methods:
```

Comment:
"should it be "To integrate with **the** API Mediation Layer"?

Reply:
API Mediation Layer is referred to just API Mediation Layer or API ML throughout the doc without 'the'.